src/unix_term.rs

```diff
@@ -94,6 +94,15 @@ impl Input<BufReader<fs::File>> {
     }
 }
 
+impl<T: Read> Read for Input<T> {
```

Comment:
Would prefer to avoid increasing the scope of this PR, please submit this separately after we merge the other changes here?

Reply:
Sure, removed.